### DISCUSSION TITLE
oufile导出parquet文件

### DISCUSSION THREAD

**Discussion by tcsll** [apache/doris#12414] [A - General / Q&A] [answered]
outfile通过s3协议导出或者通过hdfs导出parquet文件，均提示
> 1105 - errCode = 2, detailMessage = Parquet Writer is not supported yet!
1. schema已经配好了
2. 导出csv文件可以，parquet不行
3. 使用的版本是1.1.1
4. 协议与broker方式均尝试

**Comment by feng-kui** [comment]
我今天也遇到

**Reply by tcsll** [reply]
解决了哈

**Comment by LemonLiTree** [comment, marked as answer]
关闭向量化试试，导出之后查询下hive看数据有没有问题，反馈下结果谢谢

**Reply by tcsll** [reply]
在session级别set enable_vectorized_engine = false后可以成功导出parquet了,
但是我在be.conf中添加配置发现不生效，不懂是不是因为1.1.1的版本不支持，还是其他什么因素。
不过没关系，至少问题解决了，谢谢～

**Reply by LemonLiTree** [reply]
be.conf中添加配置不生效的问题我验证下吧

**Reply by tcsll** [reply]
好的谢谢～

**Reply by LemonLiTree** [reply]
问下be.conf 你配置的哪个参数不生效？

**Reply by tcsll** [reply]
我在be.conf中配置了enable_vectorized_engine_session = false

**Reply by dataalive** [reply]
没有enable_vectorized_engine_session这个参数吧，关闭向量化是session参数，set global enable_vectorized_engine = false 就持久化生效了。
